### PR TITLE
Improve flow navigation

### PR DESCRIPTION
**Pull request by lapostoj**

### CODE REVIEW

**Comment by lapostoj** [comment]
Yep, to be able to use on the "done" screens and have something like that
image

**Comment by Tankenstein** [comment]
@lapostoj @oliverviljamaa I'm wondering if it would be nicer to define a constant (oo maybe a symbol) that we export, like 

```jsx
<FlowNavigation activeStep={FlowNavigation.DONE} />
```

**Comment by lapostoj** [comment]
I was thinking about it but could it be that it won't always be in a "DONE" context? 
In itself the component doesn't have to know anything about that I would say 🤔 

Also extra small one, objections to hide the bottom border the same way as the one near the "x"?

**Comment by Tankenstein** [comment]
Can you also add something about this to docs?

File: src/flowNavigation/FlowNavigation.js [hunk 1]
```diff
@@ -23,8 +23,8 @@ const BackArrow = () => (
   </svg>
 );
 
-const FlowNavigation = ({ steps, activeStep, avatarUrl, profileType, onClose, onGoBack }) => (
-  <div className="tw-flow-navigation">
+const FlowNavigation = ({ steps, activeStep, avatarUrl, done, profileType, onClose, onGoBack }) => (
+  <div className={`tw-flow-navigation ${done ? 'tw-flow-navigation__done' : ''}`}>
```
Comment (Tankenstein) [inline src/flowNavigation/FlowNavigation.js]: Since it's a modifier, use `--done` not `__done`

**Pull request merged**